README.md

```diff
@@ -45,11 +47,12 @@ The instance qualifier is an open source command line tool that automates benchm
 * The CLI creates a CloudFormation stack with a series of resources during the run and deletes the stack at the end by default. Resources include:
   * **A VPC + Subnet + Internet Gateway**: used to launch instances. Note that they are **only created if you don't specify `vpc`/`subnet` flags or provide invalid ones**
   * **A Security Group**: same as the default security group when you create one using AWS Console.  It has an inbounding rule which opens all ports for all traffic and all protocols, but the source must be within the same security group. With this rule, the instances can access the bucket, but won't be affected by any other traffic coming outside of the security group
-  * **An IAM Role**: attached with AmazonS3FullAccess policy to allow instances to access the bucket without AWS credential configuration
+  * **An IAM Role**: attached with AmazonS3FullAccess and CloudWatchAgentServerPolicy policies to allow instances to access the bucket and emit CloudWatch metrics, respectively
   * **Launch Templates**: used to launch auto scaling group and instances
   * **An Auto Scaling Group**: the reason we use auto scaling group to manage all instances is that an one-time action can be scheduled to terminate all instances in the group after timeout to ensure the user is not excessively charged
   * **EC2 Instances**
-* An S3 bucket is also created and won't be deleted after the run, which contains the raw data if deep dive is needed
+* An **S3 bucket** containing the raw data of an Instance-Qualifier run is also created; however, this artifact is persisted by default
```

Comment:
The nittiest of nits, but the bolding of 'S3 bucket' should include the 'An' to match the other lines. Or probably better the bold should be removed from the 'A' and 'An's on the other lines

Reply:
👍🏼  I like attention to detail! will update in next revision; holding off on this merge as I need the other PR to be reviewed/merged first